src/lib/utils/theme/typography/Typography.ts

```diff
@@ -15,43 +15,37 @@ export const typographyOptions: (
     fontSize: '24px',
     fontWeight: 700,
     lineHeight: '28px',
-    marginTop: '28px',
   },
   headline2: {
     ...defaultStyles,
     fontSize: '20px',
     fontWeight: 400,
     lineHeight: '24px',
-    marginTop: '24px',
   },
   headline3: {
     ...defaultStyles,
     fontSize: '18px',
     fontWeight: 700,
     lineHeight: '20px',
-    marginTop: '20px',
   },
   body1: {
     ...defaultStyles,
     fontSize: '16px',
     fontWeight: 400,
     lineHeight: '24px',
-    marginTop: '24px',
```

Comment:
In particular, having this top margin on body1 caused each dropdown menu item to also get the margin, which looked weird. I suspect this have similar effects on other components, so it seems like this is the wrong place to apply the margin.